### DISCUSSION TITLE
There is always one of three node that goes down abnormally when importing data

### DISCUSSION THREAD

**Discussion by rvpoochen** [apache/doris#31180] [A - General / Q&A]
doris版本是2.0.4，1台fe，3台be， 标准部署部署，4台服务器系统都是ubuntu 22.04，使用insert into 批量导入1000条事件数据，导入1小时左右后，有台be就会宕机，而且一致是这台be宕机。

be.out 输出如下：
start time: 2024年 02月 20日 星期二 18:47:31 CST
INFO: java_cmd /usr/local/jdk1.8.0_211/bin/java
INFO: jdk_version 8
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/tomato/apache-doris-2.0.4-bin-x64/be/lib/java_extensions/preload-extensions/preload-extensions-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/tomato/apache-doris-2.0.4-bin-x64/be/lib/java_extensions/java-udf/java-udf-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/tomato/apache-doris-2.0.4-bin-x64/be/lib/hadoop_hdfs/common/lib/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
*** Query id: d2daf81941d64e9c-808ca8191fa3f313 ***
*** tablet id: 0 ***
*** Aborted at 1708453766 (unix time) try "date -d @1708453766" if you are using GNU date ***
*** Current BE git commitID: 003a815b63 ***
*** SIGSEGV address not mapped to object (@0x14) received by PID 4844 (TID 6218 OR 0x7f0e5eb36640) from PID 20; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/src/common/signal_handler.h:417
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 4# 0x00007F11D0A3E520 in /lib/x86_64-linux-gnu/libc.so.6
 5# doris::TFunction::~TFunction() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/Types_types.cpp:2895
 6# doris::TExprNode::~TExprNode() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/Exprs_types.cpp:2404
 7# doris::TExpr::~TExpr() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/Exprs_types.cpp:3213
 8# doris::TUnionNode::~TUnionNode() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/PlanNodes_types.cpp:13860
 9# doris::TPlanNode::~TPlanNode() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/PlanNodes_types.cpp:15890
10# doris::TPlan::~TPlan() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/PlanNodes_types.cpp:17082
11# doris::TExecPlanFragmentParamsList::~TExecPlanFragmentParamsList() at /home/zcp/repo_center/doris_release/doris/gensrc/build/gen_cpp/PaloInternalService_types.cpp:5062
12# doris::PInternalServiceImpl::_exec_plan_fragment_impl(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, doris::PFragmentRequestVersion, bool) in /home/tomato/apache-doris-2.0.4-bin-x64/be/lib/doris_be
13# doris::PInternalServiceImpl::_exec_plan_fragment_in_pthread(google::protobuf::RpcController*, doris::PExecPlanFragmentRequest const*, doris::PExecPlanFragmentResult*, google::protobuf::Closure*) at /home/zcp/repo_center/doris_release/doris/be/src/service/internal_service.cpp:305
14# doris::WorkThreadPool<false>::work_thread(int) at /home/zcp/repo_center/doris_release/doris/be/src/util/work_thread_pool.hpp:160
15# execute_native_thread_routine at ../../../../../libstdc++-v3/src/c++11/thread.cc:84
16# start_thread at ./nptl/pthread_create.c:442
17# 0x00007F11D0B22850 at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:83

请问下如何排查这类问题呢，

**Comment by yagagagaga** [comment]
Can you show me the SQL?
You can find the sql via `grep 'd2daf81941d64e9c-808ca8191fa3f313' ${FE_HOME}/log/fe.audit.log`
